app/(main)/store/item-manage/[id]/_services/getItemList.ts

```diff
@@ -0,0 +1,36 @@
+import { axiosInstance } from '@/app/_services/apiClient';
+
+export type ResponseItemTypes = {
+  itemId: number;
+  storeId: number;
+  itemName: string;
+  stock: number;
+  discountPrice: number;
+  originalPrice: number;
+  description: string;
+  image: string;
+  storeName: string;
+  storeCloseTime: string;
+  storeAddress: StoreAddressType;
+};
+
+type StoreAddressType = {
+  name: string;
+  xCoordinate: number;
+  yCoordinate: number;
+};
```

Comment:
p5: 따로 사용하는 부분이 없는 것 같아서 이렇게 해도 상관은 없을 것 같아서 리뷰 남깁니다..!

```suggestion
  storeAddress: {
  name: string;
  xCoordinate: number;
  yCoordinate: number;
};
};